humanoid_league_transform/src/humanoid_league_transform/transformer.py

```diff
@@ -208,7 +206,10 @@ def _callback_goalposts(self, msg: GoalPostInImageArray):
         for goal_post_in_image in msg.posts:
             # Check if footpoint is not in the bottom area of the image, to filter out goal posts without visible footpoint
             image_vertical_resolution =  self._camera_info.height / max(self._camera_info.binning_y, 1)
-            if goal_post_in_image.foot_point.y < image_vertical_resolution - self._goalpost_footpoint_out_of_image_threshold:
+            # Check if post is not going out of the image at the bottom
+            if self._check_if_not_bottom_of_the_image(
+                    goal_post_in_image.foot_point.y,
+                    rospy.get_param("~goalposts/footpoint_out_of_image_threshold", 0.8)):
```

Comment:
I think the `rospy.get_param` should be done in the constructor to avoid multiple calls to the parameter server